src/openms/include/OpenMS/METADATA/PeptideHit.h

```diff
@@ -57,6 +57,64 @@ namespace OpenMS
   {
 public:
 
+  struct FragmentAnnotation
+  {
+    String annotation;  // e.g. [alpha|ci$y3-H2O-NH3]5+
```

Comment:
wow, what magic syntax is this? could this be documented somewhere what these individual symbols mean?